pkg/crc/preflight/preflight_linux.go

```diff
@@ -118,6 +91,19 @@ func getPreflightChecks(experimentalFeatures bool) []Check {
 	checks = append(checks, genericPreflightChecks[:]...)
 	checks = append(checks, nonWinPreflightChecks[:]...)
 	checks = append(checks, libvirtPreflightChecks[:]...)
+	checks = append(checks, getDistroSpecificPreflights()...)
 
 	return checks
 }
+
+func getDistroSpecificPreflights() []Check {
+	releaseFile := crcos.GetReleaseFile()
+	distro, err := crcos.GetOsReleaseID(releaseFile)
```

Comment:
This should be just `distro, err := crcos.GetOsReleaseID()`, but I don't think this is enough as an API, in some cases you will also need the distro version. I think I'd add `func GetOSRelease() *OSRelease ` and `func GetOSReleaseFromBytes([]bytes) *OSRelease` (for tests)

Reply:
done